source/features/search/ui/TransactionSearchResult.tsx

```diff
@@ -37,14 +40,20 @@ export const TransactionSearchResult = () => {
           return <LoadingSpinner className={styles.loadingSpinnerMargin} />;
         } else if (transactionSearchResult) {
           return (
-            <TransactionInfo
-              dontLinkToTransaction
-              navigation={navigation?.actions}
-              networkBlockHeight={networkInfo.store.blockHeight}
-              title={translate('transaction.transactionLabel')}
-              showDetails
-              {...transactionSearchResult}
-            />
+            <>
```

Comment:
i think you can remove the React Fragment wrapper here as the TX info is the only component to render

Reply:
Done